jsonnet/kube-prometheus/main.libsonnet

```diff
@@ -70,7 +70,7 @@ local utils = import './lib/utils.libsonnet';
       image: $.values.common.images.grafana,
       prometheusName: $.values.prometheus.name,
       // TODO(paulfantom) This should be done by iterating over all objects and looking for object.mixin.grafanaDashboards
-      dashboards: $.nodeExporter.mixin.grafanaDashboards + $.prometheus.mixin.grafanaDashboards + $.kubernetesControlPlane.mixin.grafanaDashboards + $.alertmanager.mixin.grafanaDashboards,
```

Comment:
why alertmanager dashboards are removed?

Reply:
fixed it by including the upstream commit that adds the dashboards.